content/chainguard/administration/manage-chainctl-config.md

```diff
@@ -47,14 +47,13 @@ Flags:
   -h, --help   help for config
 
 Global Flags:
-      --api string                   The url of the Chainguard platform API. (default "https://console-api.enforce.dev")
-      --audience string              The Chainguard token audience to request. (default "https://console-api.enforce.dev")
-      --config string                A specific chainctl config file.
-      --console string               The url of the Chainguard platform Console. (default "https://console.enforce.dev")
-      --issuer string                The url of the Chainguard STS endpoint. (default "https://issuer.enforce.dev")
-  -o, --output string                Output format. One of: ["", "json", "id", "table", "terse", "tree", "wide"]
-      --timestamp-authority string   The url of the Chainguard Timestamp Authority endpoint. (default "https://tsa.enforce.dev")
-  -v, --v int                        Set the log verbosity level.
+      --api string        The url of the Chainguard platform API. (default "https://console-api.enforce.dev")
```

Comment:
Did you rerun the command? Just checking it's updated there as well.

Reply:
Haha, I actually didn't at first then realized I probably ought to.This is the command's output as of ~yesterday.

`console-api.enforce.dev` and `issuer.enforce.dev` aren't going to be updated, at least not any time soon